projection/core/src/multi-jvm/scala/com/lightbend/lagom/internal/projection/ProjectionRegistrySpec.scala

```diff
@@ -207,10 +240,6 @@ class ProjectionRegistrySpec extends ClusteredMultiNodeUtils(numOfNodes = 3) wit
 
     projectionRegistry.registerProjection(projectionName, tagNames, workerProps, runInRole)
 
-    tagNames.foreach { tagName =>
-      projectionRegistry.startWorker(WorkerCoordinates(projectionName, tagName))
-    }
-
```

Comment:
This invocation to `startWorkers` was a leftover falsifying some tests. The new feature in this PR surfaced this as a bug.